langtools/src/share/classes/com/sun/tools/doclets/internal/toolkit/resources/stylesheet.css

```diff
@@ -572,3 +588,4 @@ div.block div.block span.interfaceName {
 div.contentContainer ul.blockList li.blockList h2{
     padding-bottom:0px;
 }
+
```

Comment:
Now there is an extra blank line. There should be a way so the end of the file is no different from the original.

Reply:
@psoujany Can you fix this, please?